commands/audit/jas/common.go

```diff
@@ -315,3 +297,21 @@ func SetAnalyticsMetricsDataForAnalyzerManager(msi string, technologies []coreut
 		}
 	}
 }
+
+func CreateScannerTempDirectory(scanner *JasScanner, scanType string) (string, error) {
+	if scanner.TempDir == "" {
+		return "", errors.New("scanner temp dir cannot be created in an empty base dir")
+	}
+	rand.Seed(uint64(time.Now().UnixNano()))
+	randomString := ""
+	for i := 0; i < 4; i++ {
+		randomDigit := rand.Intn(10)
+		randomString += fmt.Sprintf("%d", randomDigit)
+	}
+	scannerTempDir := scanner.TempDir + "/" + scanType + "_" + randomString
```

Comment:
```suggestion
	scannerTempDir := scanner.TempDir + "/" + scanType + "_" + strconv.FormatInt(time.Now().Unix(), 10)
```
you can just use the timestamp, what do you think?